catalog/content/core/content-core-catalogerplugin/src/main/java/ddf/content/plugin/cataloger/CatalogContentPlugin.java

```diff
@@ -280,18 +276,63 @@ private Metacard generateMetacard(ContentItem contentItem, MimeType mimeType, St
                         LOGGER.debug("Unable to retrieve Security Manager.", e);
                     }
 
-                    if (uri != null) {
-                        //Setting the non-transformer specific information not including creation and modification dates/times
-                        contentMetacard.setAttribute(new AttributeImpl(Metacard.RESOURCE_URI, uri));
-                        contentMetacard.setAttribute(new AttributeImpl(Metacard.RESOURCE_SIZE,
-                                String.valueOf(size)));
+                    if (generatedMetacard instanceof DynamicMetacard) {
+                        LOGGER.debug("Adding fields to dynamic metacard of type {}", generatedMetacard.getContentTypeName());
+                        DynamicMetacard metacard = (DynamicMetacard) generatedMetacard;
+
+                        // set the user name
+                        metacard.setAttribute(Metacard.POINT_OF_CONTACT, name);
```

Comment:
Agreed - changed to set to empty string or the name (though I think setting it to null is allowed, it's just who pulls it out and doesn't check that would be the problem).